src/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  margin-top: 0;
+}
+
+#header {
+  width: 300px;
+  height: 100px;
+  background-color: blue;
+  margin-left: auto;
+  margin-right: auto;
+}
+
```

Comment:
these styles are repeated many times
```
  width: 300px;
  height: 100px;
  margin-left: auto;
  margin-right: auto;
```
 you can write them like this
```
#header, .important, ... {
  width: 300px;
  height: 100px;
  margin: 0 auto;
}
```
also, you can write the same hover